crates/jstz_cli/src/sandbox/daemon.rs

```diff
@@ -228,7 +230,7 @@ fn start_sandbox(cfg: &Config) -> Result<(OctezThread, OctezThread, OctezThread)
 pub fn main(cfg: &mut Config) -> Result<()> {
     // 1. Check if sandbox is already running
     if cfg.sandbox.is_some() {
-        return Err(anyhow::anyhow!("Sandbox is already running!"));
+        bail_user_error!("Sandbox is already running!");
```

Comment:
```suggestion
        bail_user_error!("The sandbox is already running!");
```

Reply:
Fixed 👍 (also changed the `sandbox stop` error to match)